tests/acceptance/acceptancesuite.js

```diff
@@ -160,6 +164,132 @@ test(`Facets load on the page, and can affect the search`, async t => {
   await t.expect(actualResultsCount).eql(initialResultsCount);
 });
 
+test(`static filterboxes`, async t => {
```

Comment:
I'm having a hard time understanding which pieces of functionality these tests are testing. Is it possible to split these up into multiple tests where each test is testing one thing and the functionality to be tested is included in the test title?

Reply:
I can use more words

Reply:
actually, I think I can split up the radius_filter and static_filter testing! it's unlikely these two will affect each other, though they still both need to be tested despite being on the same component